src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -0,0 +1,344 @@
+#include "SX1280_Regs.h"
+#include "SX1280_hal.h"
+#include "SX1280.h"
+
+SX1280Hal hal;
+/////////////////////////////////////////////////////////////////
+SX1280Driver *SX1280Driver::instance = NULL;
+//InterruptAssignment_ InterruptAssignment = NONE;
```

Comment:
Do we need the commented out stuff at the top here (not the steps for startup, that stuffs good, but the lines before)?

Reply:
lets remove